connect_four.py

```diff
@@ -1,8 +1,12 @@
 from enum import Enum
 
 class ColorEnum(Enum):
-        Yellow = "Y"
-        Red = "R"
+    Yellow = "Y"
```

Comment:
NIT: convention for enum names are uppercase. See:

https://zetcode.com/python/enum/#:~:text=AdvertisementsPython%20enum&text=By%20convention%2C%20enumeration%20names%20begin,or%20with%20the%20functional%20API.